sleap/config/suggestions.yaml

```
@@ -175,6 +175,12 @@ main:
         type: double
         default: 0.1
         range: 0.1,1.0
+    
+    "max point displacement":
+      - name: per_video
```

Perhaps we should give this a more accurate/descriptive name such as:
```suggestion
      - name: threshold
```